assets/stylesheets/mobile/mobile.scss

```diff
@@ -30,6 +30,10 @@
   .tc-composer-input {
     padding-right: 75px;
   }
+  .autocomplete {
+    bottom: 40px;
+    top: unset !important;
```

Comment:
This is always fishy to me. Is this necessary?

Reply:
Yeah, it is fishy. `autocomplete` uses `top` to set the position. By overriding `top`, and using a correct `bottom` value, the autocomplete box goes to the correct spot. I just realized that we own autocomplete.js so maybe I can fix a bug in there. 😢

Reply:
@eviltrout I found the fix! `treatAsTextarea` fixes the position. Removed the css.

Reply:
awesome